ipfs/client.go

```diff
@@ -0,0 +1,167 @@
+/*
```

Comment:
Maybe split the package to `ipfs/client/client.go`, so that it can be imported by third party projects without dependency on containerd pkgs

Reply:
Moved the package to ipfs/client/client.go